src/Twilio/Clients/TwilioRestClient.cs

```diff
@@ -92,6 +107,11 @@ public Response Request(Request request)
             try
             {
                 response = HttpClient.MakeRequest(request);
+                if(LogLevel == "debug")
+                {
+                    Console.WriteLine("response.status: " + response.StatusCode);
```

Comment:
Is the console the best place to log?

Reply:
After some investigation I think the console is the best place for now. If it becomes an issue later on then we can look into using a 3rd party logging library or other suggested solutions.